src/Core/src/Hosting/Dispatching/AppHostBuilderExtensions.cs

```diff
@@ -29,6 +34,99 @@ public static MauiAppBuilder ConfigureDispatching(this MauiAppBuilder builder)
 			return builder;
 		}
 
+		public static MauiAppBuilder ConfigureEnvironmentVariables(this MauiAppBuilder builder)
+		{
+			IDictionary environmentVariables = Environment.GetEnvironmentVariables();
+
+#if !NETSTANDARD
+			// For Android we read the environment variables from a text file that is written to the device/emulator
+			// If the file not exists, we will use the default environment variables which is less stable
+			if (OperatingSystem.IsAndroid() && System.IO.File.Exists("/data/local/tmp/ide-launchenv.txt"))
```

Comment:
@jonathanpeppers given this would be in the startup path, should we limit where we try and check for this file?  Eg: Should we only check if the debugger is attached? or only in a debug build?

Reply:
We added support for "fast deploying" environment variables in:

* https://github.com/xamarin/monodroid/pull/1547

Using the `@(AndroidEnvironment)` item group.

This does look like a security concern as-is, anyone can write to `/data/local/tmp/` and you can set an environment variable to specify where native libraries are loaded from. It would allow code to be injected in apps.